root/artist/header.tt

```diff
@@ -1,10 +1,10 @@
-<div class="artistheader">
-    <h1>[% link_entity(artist) %]</h1>
+[% header = artist.type_name _ 'icon' %]
+<div class = "artistheader [% header.lower() %]">
```

Comment:
You don't need the temporary variable, actually. You can just do this:
```
<div class="artistheader [% artist.type_name.lower() _ '-icon' %]">
```

Note: Remove the spaces around the `=` sign, as I did there. Also, in my previous comment I included a hyphen in `-icon`, which is what I meant by easier to read. I think variables consisting of multiple words should usually always use underscores/hyphens, though I know the current CSS isn't consistent about that.